### PR TITLE
Add Home tab to mobile nav menu

### LINKED ISSUE #370: [mobile][HAMBURGER MENU] Please add "HOME" as the first button
Please add link for HOME in mobile menu:

image

### PR DESCRIPTION
**Pull request by Brandt-Bowling**
Screenshot 2024-08-20 at 8 48 56 PM

fixes #370

### CODE REVIEW

**Review by thedomcab — APPROVED** [review]

File: frontend/src/components/header.js [hunk 1]
```diff
@@ -7,13 +7,14 @@ import Link from "./Link";
...
 const Header = () => {
   const browseList = [
-    { id: 1, title: "About", route: "/about" },
-    { id: 2, title: "Connect", route: "/connect" },
-    { id: 3, title: "Next Steps", route: "/next-steps" },
-    { id: 4, title: "Events", route: "/events" },
-    { id: 5, title: "Watch", route: "/watch" },
-    { id: 6, title: "Give", route: "/give" },
-    { id: 7, title: "I'm New", route: "/new" },
+    { id: 1, title: "Home", route: "/" },
```
Comment (xumaple) [inline frontend/src/components/header.js:10]: We shouldn't be writing these ids in manually, that makes it hard for when you want to add another page in, like you just experienced. 

Later on, where it maps through these lists, the `map` function automatically gives you an index if you want it as you're iterating through the array. Can we use that please? and change for locationsList as well?

Comment (thedomcab) [reply frontend/src/components/header.js:10]: this is the issue that react opposes using index: https://www.wisdomgeek.com/development/web-development/react/why-should-we-not-use-index-as-key-in-react-lists/#:~:text=To%20start%20with%2C%20array%20indexes,sibling%20components%20and%20inconsistent%20UI.  this is one of many resources that agree about not using indexes

Comment (Brandt-Bowling) [reply frontend/src/components/header.js:10]: Is there a better mechanism aside from hard coding an id? It seems like that's the norm?

Comment (Brandt-Bowling) [reply frontend/src/components/header.js:10]: > this is the issue that react opposes using index: https://www.wisdomgeek.com/development/web-development/react/why-should-we-not-use-index-as-key-in-react-lists/#:~:text=To%20start%20with%2C%20array%20indexes,sibling%20components%20and%20inconsistent%20UI.  this is one of many resources that agree about not using indexes
> 

Would it be better to use the page title as the key?

Comment (thedomcab) [reply frontend/src/components/header.js:10]: yes. you can use title.  as long as the keys are unique.

**Review by thedomcab — APPROVED** [review]

**Pull request merged**